CHANGELOG.md

```diff
@@ -2,13 +2,16 @@
 
 All notable changes to this project will be documented in this file.
 
-## [unreleased]
+## [2.0.0] - 2023-11-25
```

Comment:
```suggestion
## 2.0.0 - 2023-11-25
```
small thing, we don't add brackets to the version in the main repo, i guess we should do the same here?

Reply:
I believe the changelog is automated with git-cliff. I bet we have to change/create the template in order to have the changelog as you suggested

Reply:
ah, if git-cliff generates it like this it's not a problem for me then, thought this was just a typo

Reply:
Actually, we have control over the template, so I've removed them as per your suggestion.